src/services/auth.service.js

```diff
@@ -55,3 +57,21 @@ export function login({ signature, email, address }) {
     address,
   });
 }
+
+export function addUserToMailchimp(email) {
+  const data = {
+    u: import.meta.env.VITE_MAILCHIMP_USER_ID,
+    id: import.meta.env.VITE_MAILCHIMP_LIST_ID,
+    EMAIL: email,
+    "group[381006][8]": 8,
```

Comment:
Shouldn't this also be a environment variable like `VITE_MAILCHIMP_GROUP_ID` since this is temporal and it'd be useful to change the group ID without changing the code itself?

Reply:
Sure I'll do that

Reply:
👍🏼